client/app/email-domain/email/account/migrate/email-domain-email-account-migrate.html

```diff
@@ -1,207 +1,198 @@
-                            </select>
-                            <span class="oui-icon oui-icon-chevron-down" aria-hidden="true"></span>
-                        </div>
-                    </div>
-                    
-                    <!-- Destination Email -->
-                    <div class="oui-box mb-5" data-ng-if="ctrl.migrate.destinationService && !ctrl.loaders.isWaitingForDestinationEmails && !ctrl.loaders.isWaitingForMigrationCheck && !ctrl.shouldDisplayCheckMigrationsErrors">
-                        <!-- Has available email address -->
-                        <div data-ng-if="ctrl.migrate.destinationEmail">
-                    
-                            <p data-ng-bind-html="tr('email_tab_modal_migrate_available_email_selected', ctrl.migrate.destinationEmail)"></p>
-                            <p data-ng-switch="ctrl.remainingAvailableEmails && ctrl.remainingAvailableEmails > 0">
-                                <span data-ng-switch-when="true" data-ng-bind-html="tr('email_tab_modal_migrate_remaining_available_emails', ctrl.remainingAvailableEmails.toString())"></span>
-                                <span data-ng-switch-default data-i18n-static="email_tab_modal_migrate_no_remaining_available_emails"></span>
-                            </p>
-                        </div>
-                    
-                        <!-- No available email address -->
-                        <div data-ng-if="!ctrl.migrate.destinationEmail">
-                            <div class="alert alert-info" role="alert" data-i18n-static="email_tab_modal_migrate_alert_configure_me">
-                            </div>
-                    
-                            <p data-ng-bind-html="tr('email_tab_modal_migrate_no_available_email', ctrl.migrate.destinationService)"></p>
-                            <p data-ng-if="ctrl.migrate.serviceType === 'EMAIL PRO'">
-                                <a href="/#/configuration/email_pro/{{::ctrl.migrate.destinationService}}?tab=ACCOUNT" class="oui-button oui-button_primary"
-                                    data-i18n-static="email_tab_modal_migrate_order_new_emails_button"></a>
-                            </p>
-                            <p data-ng-if="ctrl.migrate.serviceType === 'HOSTED EXCHANGE'">
-                                <a href="/#/configuration/exchange_hosted/{{::ctrl.migrate.destinationService}}/{{::ctrl.migrate.destinationService}}?tab=ACCOUNT"
-                                    class="oui-button oui-button_primary" data-i18n-static="email_tab_modal_migrate_order_new_emails_button"></a>
-                            </p>
-                        </div>
-                    </div>
-                </div>
-            </fieldset>
+    <p>
+        <a class="oui-link oui-link_icon" href="{{::ctrl.allGuides}}" target="_blank">
+            <span data-i18n-static="email_tab_modal_migrate_configuration_guides_button"></span>
```

Comment:
accessibility concern: could you provide information about opening in new window.

Reply:
This could be a general concern for all our external links. What has already been done or can been done ?